.github/workflows/release-ts-api-package.yml

```diff
@@ -18,25 +25,20 @@ jobs:
 
       - name: Set ENV
         run: |
-          # extracting parachain version and worker version from release tag
-          PARACHAIN_DOCKER_TAG=$(echo ${{inputs.release-tag}} | cut -d'-' -f1 | sed 's/p/v/')
-          echo "PARACHAIN_DOCKER_TAG=$PARACHAIN_DOCKER_TAG" >> $GITHUB_ENV
-
-          WORKER_DOCKER_TAG=$(echo ${{inputs.release-tag}}  | sed 's/.*w/v/;s/-.*//')
-          echo "WORKER_DOCKER_TAG=$WORKER_DOCKER_TAG" >> $GITHUB_ENV
+          echo "DOCKER_TAG=${{inputs.docker-tag}}" >> $GITHUB_ENV
 
       - name: Pull litentry image optionally
         run: |
           docker pull parity/polkadot
-          docker pull litentry/litentry-worker:$WORKER_DOCKER_TAG
-          docker pull litentry/litentry-cli:$WORKER_DOCKER_TAG
-          docker pull litentry/litentry-parachain:$PARACHAIN_DOCKER_TAG
+          docker pull litentry/litentry-worker:$DOCKER_TAG
+          docker pull litentry/litentry-cli:$DOCKER_TAG
+          docker pull litentry/litentry-parachain:$DOCKER_TAG
 
       - name: Re-tag docker image
         run: |
-          docker tag litentry/litentry-worker:$WORKER_DOCKER_TAG litentry/litentry-worker:latest
-          docker tag litentry/litentry-cli:$WORKER_DOCKER_TAG litentry/litentry-cli:latest
-          docker tag litentry/litentry-parachain:$PARACHAIN_DOCKER_TAG litentry/litentry-parachain:latest
+          docker tag litentry/litentry-worker:$DOCKER_TAG litentry/litentry-worker:latest
+          docker tag litentry/litentry-cli:$DOCKER_TAG litentry/litentry-cli:latest
+          docker tag litentry/litentry-parachain:$DOCKER_TAG litentry/litentry-parachain:latest
```

Comment:
these changes might be adjusted once we[ resolve polling ](https://litentry.slack.com/archives/C05328L0GAF/p1716829164458979)